### PR TITLE
Doc: Uses more backquotes in documents

### PR DESCRIPTION
**Pull request by hnakamur**

### CODE REVIEW

File: doc/installing.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ sudo apt update
 sudo apt install acl attr autoconf automake dnsmasq-base git golang libacl1-dev libcap-dev liblxc1 liblxc-dev libsqlite3-dev libtool libudev-dev liblz4-dev libuv1-dev make pkg-config rsync squashfs-tools tar tcl xz-utils ebtables
 ```
 
-There are a few storage drivers for LXD besides the default directory driver.
+There are a few storage drivers for LXD besides the default `directory` driver.
```
Comment (ru-fu) [inline doc/installing.md:27]: This one should be either "the default directory driver" (as is) or "the default `dir` driver". See https://linuxcontainers.org/lxd/docs/master/reference/storage_dir/

File: doc/instances.md [hunk 2]
```diff
@@ -673,10 +673,10 @@ The `bridged`, `macvlan` and `ipvlan` interface types can be used to connect to
 `macvlan` effectively lets you fork your physical NIC, getting a second interface that's then used by the instance.
 This saves you from creating a bridge device and virtual Ethernet device pairs and usually offers better performance than a bridge.
 
-The downside to this is that macvlan devices while able to communicate between themselves and to the outside, aren't able to talk to their parent device.
-This means that you can't use macvlan if you ever need your instances to talk to the host itself.
+The downside to this is that `macvlan` devices while able to communicate between themselves and to the outside, aren't able to talk to their parent device.
+This means that you can't use `macvlan` if you ever need your instances to talk to the host itself.
 
-In such case, a bridge is preferable. A bridge will also let you use mac filtering and I/O limits which cannot be applied to a macvlan device.
+In such case, a `bridge` is preferable. A bridge will also let you use MAC filtering and I/O limits which cannot be applied to a macvlan device.
```
Comment (ru-fu) [inline doc/instances.md:679]:
```suggestion
In such case, a `bridge` device is preferable. A bridge will also let you use MAC filtering and I/O limits which cannot be applied to a macvlan device.
```

File: doc/reference/storage_btrfs.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ The following configuration options are available for storage pools that use the
 Key                             | Type      | Default                    | Description
 :--                             | :---      | :------                    | :----------
 `btrfs.mount_options`           | string    | `user_subvol_rm_allowed`   | Mount options for block devices
-`size`                          | string    | auto (20% of free disk space, >= 5 GiB and <= 30 GiB) | Size of the storage pool when creating loop-based pools (in bytes, suffixes supported)
+`size`                          | string    | `auto` (20% of free disk space, >= 5 GiB and <= 30 GiB) | Size of the storage pool when creating loop-based pools (in bytes, suffixes supported)
```
Comment (ru-fu) [inline doc/reference/storage_btrfs.md:60]:
```suggestion
`size`                          | string    | auto (20% of free disk space, >= 5 GiB and <= 30 GiB) | Size of the storage pool when creating loop-based pools (in bytes, suffixes supported)
```
No, this implies that you can specify the string `auto`, which you can't. The default is calculated automatically.

**Review by ru-fu — CHANGES_REQUESTED** [review]
Thanks! Some small updates needed, but otherwise this looks great. :)

File: doc/installing.md (hunk 1 above)
Comment (hnakamur) [reply doc/installing.md]: I changed it to the latter.

File: doc/instances.md (hunk 2 above)
Comment (hnakamur) [reply doc/instances.md]: I added "device" as suggested.

File: doc/reference/storage_btrfs.md (hunk 3 above)
Comment (hnakamur) [reply doc/reference/storage_btrfs.md]: Oh, I see. I deleted backquotes here.

**Review by ru-fu — APPROVED** [review]
Looks good, thank you! :)

**Pull request merged**